doc/api/fs.md

```diff
@@ -2529,6 +2529,19 @@ changes:
 Asynchronous rename(2). No arguments other than a possible exception are given
 to the completion callback.
 
+This function will take the old path name of a file and rename it to the path
+name provided in the second parameter.
```

Comment:
Rather than saying "old path name" and "second parameter", please use the argument names in backticks: ``` `oldPath` ``` and ``` `newPath` ```.